package.json

```diff
@@ -15,7 +15,7 @@
     "test-unit": "yarn jest test/unit/",
     "testonly": "yarn jest --runInBand",
     "testheadless": "cross-env HEADLESS=true yarn testonly",
-    "testall": "yarn test -- --ci --forceExit && lerna run --scope @next/codemod test",
+    "testall": "yarn testonly && yarn testheadless && lerna run --scope @next/codemod test",
```

Comment:
Why is `testonly` and `testheadless` being called? Won't this run tests twice once non-headless and again headless?

Reply:
That's a mistake. I just removed the headless call.

Reply:
It doesn't look like we are using this script anywhere, it might be good to remove it as we recommend running specific tests in the [contributing doc](https://github.com/vercel/next.js/blob/canary/contributing.md)